vcx/wrappers/python3/vcx/error.py

```diff
@@ -1,4 +1,8 @@
+LIBRARY = "libvcx.so"
+from vcx.cdll import _cdll
```

Comment:
Creates the ability for an ErrorCode to have its own error_message function.  Before this string was passed in at the creation of the ErrorCode object.